debian/changelog

```diff
@@ -1,3 +1,10 @@
+hidpi-daemon (18.04.7~~alpha) bionic; urgency=low
```

Comment:
Should I change this to `focal`?

Reply:
Updating the version and codename might be a good idea depending on how much longer we believe we need to include this software.